### DISCUSSION TITLE
Getting Started Page on Default Theme Redirect to 404 Page

### DISCUSSION THREAD

**Discussion by bagusa4** [vuepress/core#1518] [General]
Im using `npm init vuepress vuepress-starter`, either i select vite or webpack bundler
Anything outside post directory like Getting Started.md pages on Default Theme Redirect to 404 Page. Is this bug, issue or what ? 

_**VuePress v2**_ Running On
> node -v : v20.11.1
> npm -v : 10.2.4
> ubuntu 22.04 lts 
> linux kernel 5

**Comment by bagusa4** [comment]
Issue fix, it's just a typo, on .md file is named get-started.md but on the link named getting-started.md. And every link on this vuepress is case-sensitive.

**Comment by Mister-Hope** [comment]
typo fixed

**Comment by joker1024567** [comment]
@Mister-Hope The problem has reappeared. Issues were encountered during use on 20250815.

The reason for the 404 error on http://127.0.0.1:8080/getting-started.html is inconsistent naming between 
```
link: /getting-started.html
``` 
and the file "get-started.md"
